extensions/2.0/Khronos/KHR_gaussian_splatting/README.md

```diff
@@ -0,0 +1,389 @@
+<!--
+Copyright 2015-2024 The Khronos Group Inc.
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+# KHR_gaussian_splatting
+
+## Contributors
+
+- Jason Sobotka, Cesium
+- Adam Morris, Cesium
+- Sean Lilley, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Adding Gaussian Splats](#adding-gaussian-splats-to-primitives)
+  - [Splat Data Mapping](#splat-data-mapping)
+  - [Extension Attributes](#extension-attributes)
+  - [Transforming Data](#transforming-gaussian-splat-data-for-gltf)
+- [Limitations](#limitations)
+- [Implementation](#implementation)
+- [Meshopt Compression](#meshopt-compression)
+- [Schema](#schema)
+- [Known Implementations](#known-implementations)
+- [Resources](#resources)
+
+## Overview
+
+Currently, PLY files serve as the de facto standard through their sheer simplicity and ubiquity. This extension aims to bring structure and conformity to the Gaussian Splat space while utilizing glTF to its fullest extent. Gaussian splats are essentially a superset of a traditional point cloud, so we approached the extension with this mindset. If the point primitive contains the extension the renderer can know to render the point primitive as Gaussian splats instead of a points. Gaussian splats are defined by a position, opacity, rotation, scale, and spherical harmonics (both diffuse and specular) . We store these values as attributes on primitives.
+
+This approach allows for an easy fallback in the event the glTF is loaded within a renderer that doesn't support Gaussian splats. In this scenario, the glTF file will render as a sparse point cloud to the user. It also allows for easy integration with existing compression like meshopt.
+
+## Adding Gaussian Splats to Primitives
+
+As Gaussian splats are defined by a position color, rotation and scale, we both map to existing attributes and define new ones. These are attached to a primitive  by defining the `extensions.KHR_gaussian_splatting` property on that mesh.
+
+### Splat Data Mapping
+
+To define a Gaussian splat, it must contain the following attributes:
+
+| Splat Data | glTF Attribute | Accessor Type | Component Type |
+| --- | --- | --- | --- |
+| Position | POSITION | VEC3 | float |
+| Color (Spherical Harmonic 0 (Diffuse) and opacity) | COLOR_0 | VEC4 | unsigned byte normalized |
+| Rotation | _ROTATION | VEC4 | float |
+| Scale | _SCALE | VEC3 | float |
```

Comment:
This table could be formatted more like the table in [EXT_mesh_gpu_instancing](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md)

![image](https://github.com/user-attachments/assets/1a866ac4-89cf-48d5-bea1-e007087e235b)